v3/plugins/redhat/dependency-analytics/0.0.13/meta.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+publisher: redhat
+name: dependency-analytics
+version: 0.0.13
+type: VS Code extension
+displayName: Dependency Analytics
+title: Insights about your application dependencies
+description: Insights about your application dependencies, Security, License compatibility and AI based guidance to choose appropriate dependencies for your application.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension
+category: Other
+firstPublicationDate: '2018-10-03'
```

Comment:
Please update `firstPublicationDate` to e.g. the date of this PR (`2019-09-24`) and do similar for the latest version.

Reply:
I can address this in the next iteration. Thanks.